internal/main.go

```diff
@@ -100,13 +111,38 @@ func Main() error {
 	forceColor := flag.Bool("force-color", false, "Forcibly enable coloring when with stdout is redirected")
 	parse := flag.Bool("parse", true, "Parses source files to deduct types; use -parse=false to work around bugs in source parser")
 	verboseFlag := flag.Bool("v", false, "Enables verbose logging output")
+	filterFlag := flag.String("f", "", "A regexp to only show traces that does NOT match its, ex: -f 'IO wait|syscall'")
```

Comment:
Also it's not clear to the reader of the help page what the regexp matches, the raw string, the package?